recipe/cross_compile_support.sh

```diff
@@ -5,12 +5,21 @@ BUILD_PLATFORM=$(conda info --json | jq -r .platform)
 if [ -f ${CI_SUPPORT}/${CONFIG}.yaml ]; then
     HOST_PLATFORM=$(cat ${CI_SUPPORT}/${CONFIG}.yaml | shyaml get-value target_platform.0 ${BUILD_PLATFORM})
     CUDA_COMPILER_VERSION=$(cat ${CI_SUPPORT}/${CONFIG}.yaml | shyaml get-value cuda_compiler_version.0 None)
-    GLIBC_VERSION=$(cat ${CI_SUPPORT}/${CONFIG}.yaml | shyaml get-value c_stdlib_version.0 2.17)
 fi
 
+# When compiling natively we use the docker image's glibc
+# We do the same now when cross-compiling by creating a
+# conda environment for host platform with the same glibc
+# as the docker glibc.
+# For eg: when cross compiling for linux-aarch64 in a
+# docker image with linux-anvil-x86_64:alma9 image, we use
+# glibc=2.28 for QEMU while still building the package for
```

Comment:
I think the comment is slightly off - alma9 would yield a glibc of 2.34 through `__glibc`; 2.28 goes with alma8.